src/main/kotlin/com/hibob/academy/employeeFeedback/service/feedback/FeedbackCreatorService.kt

```diff
@@ -0,0 +1,24 @@
+package com.hibob.academy.employeeFeedback.service.feedback
+
+import com.hibob.academy.employeeFeedback.dao.FeedbackDao
+import com.hibob.academy.employeeFeedback.model.CreateFeedbackRequest
+import com.hibob.academy.employeeFeedback.utils.getLoggedInUser
+import org.springframework.stereotype.Service
+
+@Service
+class FeedbackCreatorService(
+    private val feedbackDao: FeedbackDao
+) {
+
+    fun createFeedback(createFeedbackRequest: CreateFeedbackRequest) : Long {
```

Comment:
I expect that the loggedIn user will be passed as a parameter from the controller